clang/include/clang/Basic/AttrDocs.td

```diff
@@ -57,6 +57,15 @@ global variable or function should be in after translation.
   let Heading = "section, __declspec(allocate)";
 }
 
+def CodeModelDocs : Documentation {
+  let Category = DocCatVariable;
+  let Content = [{
+The ``model`` attribute allows you to specify a specific code model a
+global variable should be in after translation.
```

Comment:
"allows you to override the code model that accesses to the decorated global variable should be in"? Or something similar to the GCC docs for the attribute.

Reply:
Done